specs/altair/sync-protocol.md

```diff
@@ -47,34 +49,25 @@ uses sync committees introduced in [this beacon chain extension](./beacon-chain.
 
 ### Misc
 
-| Name | Value |
-| - | - |
-| `MIN_SYNC_COMMITTEE_PARTICIPANTS` | `1` |
+| Name | Value | Notes |
+| - | - | - |
+| `MIN_SYNC_COMMITTEE_PARTICIPANTS` | `1` | |
+| `SAFETY_THRESHOLD_PERIOD` | `SLOTS_PER_EPOCH * EPOCHS_PER_SYNC_COMMITTEE_PERIOD` | ~13.6 hours |
+| `UPDATE_TIMEOUT` | `SLOTS_PER_EPOCH * EPOCHS_PER_SYNC_COMMITTEE_PERIOD` | ~27.3 hours |
```

Comment:
I think one of these is wrong?
`SLOTS_PER_EPOCH * EPOCHS_PER_SYNC_COMMITTEE_PERIOD = 32 * 256 = 8192` slots = ~27.3 hours.
Did you mean having `SAFETY_THRESHOLD_PERIOD = UPDATE_TIMEOUT / 2`?

Reply:
Addressed in #2750